src/features/events/layout/EventLayout.tsx

```diff
@@ -71,10 +71,7 @@ const EventLayout: React.FC<EventLayoutProps> = ({
                 <EventTypeAutocomplete
                   onBlur={() => setEditingTypeOrTitle(false)}
                   onChange={(newValue) => {
-                    if (newValue) {
-                      // TODO: Pass null values as well, when API supports it
-                      model.setType(newValue.id);
-                    }
+                    model.setType(newValue ? newValue.id : newValue);
```

Comment:
This is fine as it is, but I would have done something like this instead, to clarify to a reader of this code that the fallback is `null`:

```suggestion
                    model.setType(newValue ? newValue.id : null);
```

Or this:

```suggestion
                    model.setType(newValue?.id ?? null);
```